index.js

```diff
@@ -1 +1,84 @@
-// implement your API here
+// ---------------------DEPENDACIES-------------------------//
+const express = require("express");
+const db = require("./data/db.js");
+const server = express();
+
+// ---------------------SERVER-------------------------//
+const port = 4000;
```

Comment:
* Recommendation
  - I would stick with the convention of using PORT as this is what heroku will use when you deploy eventually.